spring-security-oauth2/src/main/java/org/springframework/security/oauth2/common/util/SerializationUtils.java

```diff
@@ -8,6 +8,10 @@
 
 import org.springframework.core.ConfigurableObjectInputStream;
 
+/**
+ * This class should not be used since it uses unsafe deserialization via {@link ConfigurableObjectInputStream}.
+ */
+@Deprecated
```

Comment:
`SerializationUtils` is still used in certain places. We should ensure there is no further usages since `SerializationStrategy` is now preferred.

Reply:
Right, I missed several places where it's used. I'll replace `SerializationUtils` with `WhitelistedSerializationStrategy`.

I also noticed that some tests use `org.springframework.util.SerializationUtils`. I guess it can be also replaced with `WhitelistedSerializationStrategy`.